content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -270,30 +270,30 @@ Containers within the Pod see the system hostname as being the same as the confi
 `name` for the Pod. There's more about this in the [networking](/docs/concepts/cluster-administration/networking/)
 section.
 
-## Privileged mode for containers
-
-{{< note >}}
-Your {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}} must support the concept of a privileged container for this setting to be relevant.
-{{< /note >}}
-
-Any container in a pod can run in privileged mode to use operating system administrative capabilities
-that would otherwise be inaccessible. This is available for both Windows and Linux.
-
-### Linux privileged containers
-
-In Linux, any container in a Pod can enable privileged mode using the `privileged` (Linux) flag
-on the [security context](/docs/tasks/configure-pod-container/security-context/) of the
-container spec. This is useful for containers that want to use operating system administrative
-capabilities such as manipulating the network stack or accessing hardware devices.
-
-### Windows privileged containers
-
-{{< feature-state for_k8s_version="v1.26" state="stable" >}}
-
-In Windows, you can create a [Windows HostProcess pod](/docs/tasks/configure-pod-container/create-hostprocess-pod) by setting the 
-`windowsOptions.hostProcess` flag on the security context of the pod spec. All containers in these
-pods must run as Windows HostProcess containers. HostProcess pods run directly on the host and can also be used
-to perform administrative tasks as is done with Linux privileged containers.
+## Pod security settings {#pod-security}
+
+To set security constraints on Pods and containers, you use the
+`securityContext` field in the Pod specification. This field gives you
+granular control over what a Pod or individual containers can do. For example:
+
+* Drop specific Linux capabilities to avoid the impact of a CVE.
+* Force all processes in the Pod to run as a non-root user or as a specific
+  user or group ID.
+* Set a specific seccomp profile.
+* Set Windows security options, such as whether containers run as HostProcess.
+
+{{< caution >}}
+You can also use the Pod securityContext to enable
+[*Privileged mode*](/docs/concepts/security/linux-kernel-security-constraints/#privileged-containers).
+Privileged mode overrides many of the other security settings in the
+securityContext. Avoid using this setting unless you can't grant the equivalent
+permissions by using other fields in the securityContext.
+{{< /caution >}}
```

Comment:
This change drops a mention of using `windowsOptions.hostProcess` for Windows containers, and doesn't highlight that host process Windows containers are only stable since v1.26

Can we retain those details somehow? The link to https://k8s.io//docs/tasks/configure-pod-container/create-hostprocess-pod/ is worth keeping.

We could also update https://k8s.io/docs/concepts/windows/intro/ to link to https://k8s.io/docs/tasks/configure-pod-container/create-hostprocess-pod/ and then change this page to signpost readers to the relevant section of https://kubernetes.io/docs/concepts/windows/intro/ - do that if you think it overall works better.

Reply:
Alternative: write `content/en/docs/concepts/security/windows-kernel-security-constraints.md` (a very short page!)

Reply:
Added the mention back, but I like the idea of a dedicated page and would like to do a followup PR for that